front/app/components/InitiativeCards/ProposalsList.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { adopt } from 'react-adopt';
+import { isNilOrError } from 'utils/helperUtils';
+// tracks
+import { trackEventByName } from 'utils/analytics';
+import tracks from './tracks';
+
+// components
+import InitiativeCard from 'components/InitiativeCard';
+import { Spinner, Button } from '@citizenlab/cl2-component-library';
+
+// resources
+import GetInitiatives, {
+  GetInitiativesChildProps,
+} from 'resources/GetInitiatives';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// style
+import styled, { useTheme } from 'styled-components';
+import { media } from 'utils/styleUtils';
+import { rgba } from 'polished';
+import useLocale from 'hooks/useLocale';
+
+const Loading = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  ${media.biggerThanMinTablet`
+    height: calc(100vh - 280px);
+    position: sticky;
+    top: 200px;
+  `}
+
+  ${media.smallerThanMinTablet`
+    height: 150px;
+  `}
+`;
+
+const Footer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+
+  ${media.smallerThanMinTablet`
+    flex-direction: column;
+    align-items: stretch;
+    margin-top: 0px;
+  `}
+`;
+
+const ShowMoreButton = styled(Button)``;
```

Comment:
Just copy-pasted to not create extra work :)

Reply:
Maybe we should fix it in both places, then? 🙂

Reply:
Haha, I get you but I've learned to limit the scope of my work when time is limited because one time you think you're changing 1 small thing and then it cascades into more changes or you need to test again, etc. I